Eplant/views/eFP/Viewer/MaskModal.tsx

```diff
@@ -0,0 +1,68 @@
+// Import necessary dependencies from Material-UI
+import React, { useState } from 'react';
+import { Modal, Slider, Typography, Button, useTheme } from '@mui/material';
+import { EFPViewerState } from './types';
+
+// Modal component with a slider
+interface MaskModalProps {
+    state: EFPViewerState
+    onClose: () => void
+    onSubmit: (threshhold: number) => void
+}
+
+const MaskModal: React.FC<MaskModalProps> = ({ state, onClose, onSubmit }) => {
```

Comment:
nit: `React.FC` is not necessary (although not necessarily bad either).

[This article](https://www.totaltypescript.com/you-can-stop-hating-react-fc) recommends using type annotations instead.
```suggestion
const MaskModal = ({ state, onClose, onSubmit }: MaskModalProps) => {
```